ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/privacy/methods/priv/PrivFindPrivacyGroup.java

```diff
@@ -53,10 +54,9 @@ public JsonRpcResponse doResponse(final JsonRpcRequestContext requestContext) {
 
     LOG.trace("Finding a privacy group with members {}", Arrays.toString(addresses));
 
-    FindPrivacyGroupRequest findPrivacyGroupRequest = new FindPrivacyGroupRequest(addresses);
     PrivacyGroup[] response;
     try {
-      response = enclave.findPrivacyGroup(findPrivacyGroupRequest);
+      response = privateTransactionHandler.findPrivacyGroup(addresses);
```

Comment:
probably need to rename "PrivateTransactionHandler" - and make it a "PrivacyManager/Controller/Hub" ... aka god class.

Reply:
agreed, don't like this name as it's dealing with more than transactions. something controller it probably the way to go.

Reply:
done